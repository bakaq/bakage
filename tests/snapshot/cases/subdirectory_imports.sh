#!/bin/sh

bakage() {
    ../../../../build/bakage.pl "$@"
}

echo "=== Installing dependencies ==="
bakage install

echo ""
echo "=== Running test from project root ==="
scryer-prolog tests/test_my_module.pl -g "run_tests, halt."

echo ""
echo "=== Running test from subdirectory ==="
cd tests
scryer-prolog test_my_module.pl -g "run_tests, halt."
