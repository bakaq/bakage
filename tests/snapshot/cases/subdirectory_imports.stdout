=== Installing dependencies ===
Ensuring is installed: git("https://github.com/bakaq/testing.pl.git")

=== Running test from project root ===
Running tests in module [37;1muser[0m.
  test "greet succeeds" ... Hello, World!
[32;1msucceeded[0m

=== Running test from subdirectory ===
Running tests in module [37;1muser[0m.
  test "greet succeeds" ... Hello, World!
[32;1msucceeded[0m
